src/utils/sign_utils.ts

```diff
@@ -0,0 +1,30 @@
+function getTypedData(networkId: number, securityTokenAddress: string, reason: string) {
+  const typedData = {
+    types: {
+      EIP712Domain: [
```

Comment:
Maybe a comment referring to documentation in EIP712 Repo?

Reply:
Mmh maybe it would be more useful a link to our repo with this checks, I'll add it.